python/mxnet/ndarray/random.py

```diff
@@ -152,7 +152,7 @@ def normal(loc=0, scale=1, shape=_Null, dtype=_Null, ctx=None, out=None, **kwarg
                           [loc, scale], shape, dtype, ctx, out, kwargs)
 
 
-def randn(*shape, **kwargs):
+def randn(loc=0, scale=1, shape=_Null, dtype=_Null, ctx=None, out=None, **kwargs):
```

Comment:
This seems to be a breaking change: For example mx.nd.random.randn(2, 3, loc=5, scale=1) will fail now. We should add this to APIs good to break for 2.0 https://github.com/apache/incubator-mxnet/issues/9686 and postpone this.